src/test/java/capstone/mfslbackend/service/TeamServiceTest.java

```diff
@@ -160,18 +176,18 @@ public void testCreateTeamById_SuccessExistsAlready() throws IOException {
         TeamsContainer teamsContainer = new TeamsContainer();
         List<TeamsResponse> teamsResponses = new ArrayList<>();
 
-        TeamResponse teamResponse1 = new TeamResponse();
-        teamResponse1.setId(teamId);
-        teamResponse1.setName(teamName);
-        teamResponse1.setLogo(teamName);
+        TeamResponse teamResponse1 = new TeamResponse(teamId, teamName, teamName, teamName, true, teamName, teamName, 1, true);
+//        teamResponse1.setId(teamId);
```

Comment:
no need to comment it out, if youre gonna change it to use the constructor, just remove these lines